src/biogeophys/test/WaterTracerUtils_test/test_compare_bulk_to_tracer.pf

```diff
@@ -0,0 +1,127 @@
+module test_compare_bulk_to_tracer
```

Comment:
@mathewvrothstein - I added these unit tests largely for the sake of ensuring that nobody breaks the CompareBulkToTracer routine when it is modified in the future. You may want to take a quick look at these unit tests as a fairly simple example of pfunit-based unit tests, and also because you'll likely need to modify them a bit when you add the ratio argument.